### ISSUE TITLE
Create diagram of Gitpod loading process

### ISSUE THREAD

**Issue by loujaybee** [gitpod-io/website#2368]
It's important for users to know what is loaded into a Gitpod workspace (and when) so that they can update their configurations knowing that certain features might be available at different times. 

- IDE (JetBrains / VS Code)
- Dotfiles
- Prebuilds

Relates to: 
- https://github.com/gitpod-io/gitpod/issues/11063
- https://github.com/gitpod-io/website/issues/1651

Note: This was mentioned elsewhere, need to find the references. 

Related 
- [thread](https://discord.com/channels/816244985187008514/1021781957077635134/1021854049198424074)
- [thread](https://discord.com/channels/816244985187008514/1022121237750755420/1022121243371126834)

**Comment by axonasif** [comment]
Suggested diagrams:
- https://twitter.com/JPhi_Baconnais/status/1565282715988922368
- https://excalidraw.com/#room=fb10397dd93ab229efb3,uoIgyuDBiprNcS07mvyXQA

Internal slack thread: https://gitpod.slack.com/archives/C021JDXHXMF/p1663697920547039

**Comment by loujaybee** [comment]
Will assign this to @axonasif who is going to take a look at the [life of a workspace page](https://www.gitpod.io/docs/introduction/learn-gitpod/the-life-of-a-workspace). And create a draft of a refresh to the content on that page as a start. We'll de-scope from this issue: 

* Dotfiles loading process (https://www.gitpod.io/docs/configure/user-settings/dotfiles)
* Tasks loading process (https://www.gitpod.io/docs/configure/workspaces/tasks)

And pick those up in separate issues.

**Comment by loujaybee** [comment]
> wasn't meant to be closed

Yeah, indeed let's keep it open for now, as we're making progress—but it's not yet complete!
